src/commonMain/kotlin/exchange.dydx.abacus/state/manager/V4StateManagerAdaptor.kt

```diff
@@ -1449,10 +1449,19 @@ class V4StateManagerAdaptor(
     }
 
     private fun errorTrackingParams(error: ParsingError): IMap<String, Any> {
-        return iMapOf(
-            "errorType" to error.type.rawValue,
-            "errorMessage" to error.message,
-        )
+        val errorStringKey = error.stringKey
+        return if (errorStringKey != null) {
+            iMapOf(
+                "errorType" to error.type.rawValue,
+                "errorMessage" to error.message,
+                "errorStringKey" to error.stringKey,
```

Comment:
nit: we can probably reuse `errorStringKey` here (or just use `error.stringKey` everywhere)